src/main/java/com/github/webicitybrowser/spec/fetch/imp/FetchEngineImp.java

```diff
@@ -1,20 +1,22 @@
 package com.github.webicitybrowser.spec.fetch.imp;
 
-import com.github.webicitybrowser.spec.fetch.FetchEngine;
-import com.github.webicitybrowser.spec.fetch.FetchParameters;
-import com.github.webicitybrowser.spec.fetch.FetchParams;
-import com.github.webicitybrowser.spec.fetch.FetchResponse;
+import com.github.webicitybrowser.spec.fetch.*;
 import com.github.webicitybrowser.spec.fetch.connection.FetchConnection;
 import com.github.webicitybrowser.spec.fetch.connection.FetchConnectionPool;
 import com.github.webicitybrowser.spec.fetch.connection.FetchNetworkPartitionKey;
 import com.github.webicitybrowser.spec.url.URL;
 
+import java.util.LinkedList;
+import java.util.Queue;
+
 public class FetchEngineImp implements FetchEngine {
 
 	private final FetchConnectionPool connectionPool;
+	private Queue<FetchConsumeBodyAction> fetchQueue;
```

Comment:
This should be `final`

But this will need to be replaced by a call to queueAFetchTask, which will instead queue on the taskDestination, anyways